operator/internal/manifests/node_placement.go

```diff
@@ -12,6 +13,13 @@ import (
 const (
 	availabilityZoneEnvVarName = "INSTANCE_AVAILABILITY_ZONE"
 	availabilityZoneFieldPath  = "metadata.annotations['" + lokiv1.AnnotationAvailabilityZone + "']"
+	// availabilityZoneVolumeName is the name of the volume that will contain the
+	// availability zone annotation we get from DownwardAPI
+	availabilityZoneVolumeName = "az-annotation"
+	// availabilityZoneVolumeMountPath path where the volume will be mounted on the init container
+	availabilityZoneVolumeMountPath = "/etc/az-annotation"
+	// availabilityZoneVolumeFileName name of the file containing the availability zone annotation
+	availabilityZoneVolumeFileName = "az"
```

Comment:
If we only need these in the init container then adding an `Init` to the constant name might be an idea?

```suggestion
	availabilityZoneInitVolumeName      = "az-annotation"
	availabilityZoneInitVolumeMountPath = "/etc/az-annotation"
	availabilityZoneInitVolumeFileName  = "az"
```